include/libtorrent/torrent_handle.hpp

```diff
@@ -835,6 +835,11 @@ namespace libtorrent
 		void queue_position_down() const;
 		void queue_position_top() const;
 		void queue_position_bottom() const;
+		// set the new position in queue starting by 0. If given position does
```

Comment:
I believe you may need a blank line here, for the documentation tool to pick this up correctly